include/zephyr/net/net_ip.h

```diff
@@ -373,14 +387,23 @@ struct cmsghdr {
 #endif
 #endif
 
+#if defined(CONFIG_NET_SOCKETS_OFFLOAD)
```

Comment:
Should rather rely on `CONFIG_NET_NATIVE_OFFLOADED_SOCKETS`, `CONFIG_NET_SOCKETS_OFFLOAD` is a generic term, don't want to increase this for every offloaded case, it's NSOS specific.

Reply:
It should have been fixed in the last rebase.